server/src/test/java/org/opensearch/action/get/TransportGetActionTests.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.action.get;
+
+import org.opensearch.Version;
+import org.opensearch.cluster.ClusterName;
+import org.opensearch.cluster.ClusterState;
+import org.opensearch.cluster.metadata.IndexMetadata;
+import org.opensearch.cluster.metadata.Metadata;
+import org.opensearch.cluster.routing.Preference;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.core.index.Index;
+import org.opensearch.indices.replication.common.ReplicationType;
+import org.opensearch.test.OpenSearchTestCase;
+import org.junit.BeforeClass;
+
+import static org.opensearch.common.UUIDs.randomBase64UUID;
+
+public class TransportGetActionTests extends OpenSearchTestCase {
+
+    private static ClusterState clusterState;
+    private static ClusterState clusterState2;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
```

Comment:
FYI, prefer `@Before` and non-static fields. In general it is better to have instance state that is not shared between test executions to avoid problems with tests interfering with one another.

However, in this case I think you can create a `private static ClusterState clusterState(ReplicationType)` static method and then create the cluster state on-demand for each test case below. That will also solve the problem of the unhelpfully named field `clusterState2` because you can replace it with `clusterState(ReplicationType.DOCUMENT)` :)

Reply:
Thanks for the review @andrross! :) Made the changes as suggested